superset-frontend/plugins/plugin-chart-pivot-table/src/plugin/controlPanel.tsx

```diff
@@ -95,10 +95,14 @@ const config: ControlPanelConfig = {
             name: 'row_limit',
             config: {
               ...sharedControls.row_limit,
+              label: t('Cell limit'),
+              description: t('Limits the number of cells that get displayed.'),
```

Comment:
Maybe this is more accurate?
```suggestion
              description: t('Limits the number of cells that get retieved.'),
```